src/util/markdown-input.html

```diff
@@ -65,16 +73,27 @@
           notify: true
         },
         label: String,
-        previewToggle: Boolean,
-        hidePreview: {
+        _previewToggle: Boolean,
+        _hidePreview: {
           type: Boolean,
-          computed: '_hidePreview(previewToggle, disabled)'
+          computed: '_getHidePreview(_previewToggle, disabled)'
+        },
+        _toggleText: {
+          type: String,
+          computed: '_getToggleText(_previewToggle)'
         }
       },
-      _hidePreview: function(previewToggle, disabled) {
+      _getHidePreview: function(previewToggle, disabled) {
         var shouldBeHidden = !disabled && !previewToggle;
         this.toggleClass('noPreview', shouldBeHidden);
         return shouldBeHidden;
+      },
+      _getToggleText: function(active) {
+        if (active) {
```

Comment:
I personally think inline if-statements should be avoided if they do not provide a significant benefit. In this case a full if statement is equivalent in functionality and easier to read.